components/layout/header.vue

```diff
@@ -43,7 +43,7 @@
     >
       <PopoverPanel
         focus
-        class="bg-white-500 h-auto absolute top-0 inset-x-0 transition transform origin-top-right md:hidden overflow-hidden"
+        class="bg-white-500 z-10 h-auto absolute top-0 inset-x-0 transition transform origin-top-right md:hidden overflow-hidden"
```

Comment:
what do you need z for?

Reply:
so the drop-down menu does not overlap with the page content. This should also fix the partners logo overlapping it